Makefile

```diff
@@ -14,4 +14,4 @@ generate: install
 	bash scripts/generate.sh
 
 test-docker: generate
-	bash ./prism.sh
+	bash ./prism.sh $(SONAR_TOKEN)
```

Comment:
What happens when this is run locally without the env var set? I ask because I frequently run this target to test changes.

Reply:
When you run this locally, you can pass this variable in the make command itself. Should this sonar analysis be separated to a separate step so that we can run the test step locally? Here we might need to duplicate the code for running docker and commands to checkout twilio-csharpo repo and copy generated files. Can we go ahead with that approach?

Reply:
Updated the code  to run sonar analysis only when SONAR_TOKEN variable has value. In local runs this will have a value of "" and the analysis will be skipped. Tested this locally, this is working fine